CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# v1.1.0 (2022-12-15)
+
+## Features:
+* See the [bioformats2raw changelog](https://github.com/glencoesoftware/bioformats2raw/releases/tag/v0.6.0) for details of new features.
```

Comment:
I could also see a sentence stating the converted now writes OME-NGFF according to the [0.4 version of the specification](https://ngff.openmicroscopy.org/0.4/)